File: Mensuration/Triangle and Prism/Perimeter/PoT.c
#include<stdio.h>
void main(){
    int a,b,c,peri;
    printf("Enter 1st Side : ");
    scanf("%d",&a);
    printf("Enter 2nd Side : ");
    scanf("%d",&b);
    printf("Enter 3rd Side : ");
    scanf("%d",&c);
    peri=a+b+c;
    printf("Perimeter : %d",peri);
}

File: Problem Statements/Mensuration/Triangle and Prism/Perimeter/PoT.java
import java.util.Scanner;
class PoT{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter 1st Side : ");
        int a=sc.nextInt();
        System.out.println("Enter 2nd Side : ");
        int b=sc.nextInt();
        System.out.println("Enter 3rd Side : ");
        int c=sc.nextInt();
        sc.close();
        int peri=a+b+c;
        System.out.println("Perimeter : "+peri);
    }
}